module-2/backend/week_one.md

```diff
@@ -5,28 +5,92 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+GET to get info from the server
+Post to send info to the server
+DELETE to erease an entry in the server
+PUT replace entry in database
+PATCH modify an entry in the database
+
 2. What is Sinatra?
+A lightweight web server app
+
+3. What is a wireframe and why/when should we wireframe?
+Mock sup how you/client thinks a page should look. Useful to discuss the look of your site. 
+
 4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
-7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
+Model View Controller
+
+5. What's the purpose of having a `Task` model in our `TaskManager` application?
+To be able to make instances of Task. 
+
+6. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+It follows REST conventions and is also how Rails works with REST. 
+
+7. What types of variables are accessible in our view templates without explicitly passing them?
+Instance 
+
+8. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
     erb :index
   end
   ```
+  @count = 1
+
+9. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
+:locals => {:name => 'Mr. Ed'}
+
+10. What's the purpose of ERB?
+to be able to execute Ruby within an HTML view. 
+
+11. Why do I need a development AND test database?
+You dont want to be working with actual data when designing the database and site. 

```

Comment:
why do we want specifically a development and test database though? what's the difference between the two?